PubSubSpec.md

```diff
@@ -2,6 +2,8 @@
 
 This page describes the specification of the SVS-PS protocol. SVS-PS runs on top of State Vector Sync and performs the additional functions described here.
 
+**Last update to specification**: 2022-12-27
```

Comment:
(commenting on a random place because github doesn't allow comments on the commit message)

This commit does more than what the commit message says. I recommend listing all "important" changes in the commit message or splitting this commit in multiple atomic changes.